testing/firefox/APKBUILD

```diff
@@ -121,7 +121,6 @@ build() {
 		--enable-default-toolkit=cairo-gtk3 \
 		--enable-official-branding \
 		--enable-optimize="$CFLAGS" \
-		--enable-pie \
```

Comment:
You probable want to replace this with --enable-hardened and not remove it

Reply:
I think you mean `--enable-hardening`